src/App.tsx

```diff
@@ -1,14 +1,48 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
-import { TodoFilter } from './components/TodoFilter';
-import { TodoModal } from './components/TodoModal';
+import { IFilter, TodoFilter } from './components/TodoFilter';
+// import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { TodoModal } from './components/TodoModal';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filteredTodos, setFilteredTodos] = useState<Todo[]>([]);
+  const [todoId, setTodoId] = useState<number>(0);
+
+  const [filter, setFilter] = useState<IFilter>({
+    status: 'all',
+    searchTitle: '',
+  });
+
+  useEffect(() => {
+    getTodos().then((result) => setTodos(result));
```

Comment:
where is block `catch`?
btw, would be better to use async/await syntax
```suggestion
    getTodos().then(setTodos);
```